OracleIdentityGovernance/README.md

```diff
@@ -0,0 +1,128 @@
+Oracle Identity Governance (OIG) on Docker
+==========================================
+
+
+## Contents
+
+1. [Introduction](#introduction)
+2. [Prerequisites for OIG on Docker](#prerequisites-for-oig-on-docker)
+3. [Prerequisites for OIG on Kubernetes](#prerequisites-for-oig-on-kubernetes)
+4. [Installing the OIG Docker Image](#installing-the-oig-docker-image)
+5. [OIG Docker Container Configuration](#oig-docker-container-configuration)
+6. [OIG Kubernetes Configuration](#oig-kubernetes-configuration)
+7. [Appendix A: Using an Oracle Database Docker Image for OIG](#appendix-a-using-an-oracle-database-docker-image-for-oig)
+
+## Introduction
+
+This project offers Dockerfiles and scripts to build and configure an Oracle Identity Governance image based on 12cPS4 (12.2.1.4.0) release.
+Use this Docker Image to facilitate installation, configuration, and environment setup for DevOps users. 
+
+## Prerequisites for OIG on Docker
+
+The following prerequisites are required for building OIG Docker images:
+
+* A working installation of Docker 18.03 or later
+* Docker-compose 1.25.4 or higher
+* A running Oracle Database 12.2.0.1 or later. This database can be installed anywhere. The database must be a supported version for OIG as outlined in [Oracle Fusion Middleware 12c certifications](https://www.oracle.com/technetwork/middleware/fmw-122140-certmatrix-5763476.xlsx), and must meet the requirements as outlined in [About Database Requirements for an Oracle Fusion Middleware Installation](http://www.oracle.com/pls/topic/lookup?ctx=fmw122140&id=GUID-4D3068C8-6686-490A-9C3C-E6D2A435F20A).  If you require a test database, an Oracle Database Docker image can be used. For instructions on installing and running an Oracle Database Docker image refer to section [Appendix A: Using an Oracle Database Docker Image for OIG](#appendix-a-using-an-oracle-database-docker-image-for-oig) at the end of this README.
+
+### Hardware Requirements
+
+| Hardware  | Size                                              |
+| :-------- | :-------------------------------------------------|
+| RAM       | Min 16GB                                          |
+| Disk Space| Min 50GB (ensure 10G+ available in Docker Home)   |
```

Comment:
"Docker Home" is not a known term. I would change this to either `Docker Root Dir` (which is the field shown in the output of `docker info`) or something more generic like "ensure at least 10GB is available on the container image storage mount point" because other tools use other locations.